CMakeLists.txt

```diff
@@ -69,30 +15,25 @@ message( STATUS "CMAKE_CXX_COMPILER_ID: " ${CMAKE_CXX_COMPILER_ID} )
 if ( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang" )
     message( STATUS "C++ Compiler is Clang" )
     set( PPICM_HAS_CLANGXX "TRUE" )
-   # set( CMAKE_CXX_FLAGS "-stdlib=libc++ -std=gnu++11" )
-    set( CMAKE_CXX_FLAGS "-std=c++14" )
-    set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-register -Wno-empty-body -Wno-unused-value" )
-   # add_definitions( -stdlib=libc++ -std=gnu++11 )  # Or -std=c++11
+    set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wno-deprecated-register -Wno-empty-body -Wno-unused-value" )
+
 elseif ( ${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU" )
     message( STATUS "C++ Compiler is GNU")
     execute_process( COMMAND ${CMAKE_CXX_COMPILER} -dumpversion OUTPUT_VARIABLE GXX_VERSION )
     if(NOT(GXX_VERSION VERSION_GREATER 4.1))
-        set( GCC_TOO_OLD 1 )
         message( FATAL_ERROR "g++ Version < 4.1, some targets will be skipped" )
         message( WARNING "Use CC/CXX env variable to define a newer compiler, e.g. export CC=/usr/bin/gcc-4.2; export CXX=/usr/bin/g++-4.2" )
     else(NOT(GXX_VERSION VERSION_GREATER 4.1))
-        add_definitions( -std=c++14 ) # or -std=c++0x
-	    add_definitions( -O3 -g )
-        # FASTFLOW SPECIFIC	
-	    add_definitions(-DNO_DEFAULT_MAPPING) # -DBLOCKING_MODE 
-        #add_definitions( -std=c++11 ) # or -std=c++0x 
+        add_definitions( -std=c++14 -O3 -g )
+       
     endif(NOT(GXX_VERSION VERSION_GREATER 4.1))
 
 elseif ( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Intel" )
     message( STATUS "C++ Compiler is Intel" )
     message( WARNING "Compliance check not fully implemented yet" )
 endif()
 
+# GrPPI library
 include_directories("${CMAKE_SOURCE_DIR}/include")
```

Comment:
We should check all CMakes in our tree to check if there are additiona include directives